src/System.Reflection.Metadata/tests/PortableExecutable/PEReaderTests.cs

```diff
@@ -298,7 +298,25 @@ public void GetSectionData_Errors()
         }
 
         [Fact]
-        public void TryOpenAssociatedPortablePdb_Args()
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
Nit but every other place you put Desktop test before Core test. Consistency is generally a good thing as it helps the eye catch errors...